chapters.md

```diff
@@ -128,25 +128,23 @@ The `ChapterAtom` is also called a `Chapter`.
 A `Chapter` element can be used recursively. Such a child `Chapter` is called `Nested Chapter`.
 
 ### ChapterTimeStart
-A not scaled timestamp of the start of `Chapter` with nanosecond accuracy.
+The timestamp of the start of `Chapter` with nanosecond accuracy, not scaled by TimestampScale.
 For `Simple Chapters` this is the position of the chapter markers in the timeline.
 
 ### ChapterTimeEnd
-A not scaled timestamp of the end of `Chapter` with nanosecond accuracy.
-The end timestamp is used when the `EditionFlagOrdered` flag of the `Edition` is set to `true`.
+The timestamp of the end of `Chapter` with nanosecond accuracy, not scaled by TimestampScale.
 The timestamp defined by the `ChapterTimeEnd` is not part of the `Chapter`.
+The end timestamp is used when the `EditionFlagOrdered` flag of the `Edition` is set to `true`.
```

Comment:
On the one hand, you have "fought" to ensure that the `ChapterTimeEnd` is used(can be used) even without an ordered edition.
 #440
 removing the `ChapterEndTime` from the EditionFlagOrdered table.

And now is the element used if the `EditionFlagOrdered` flag of the `Edition` is set to `true`.

What is now correct?

Reply:
This is not a new sentence I added, only the text around has been moved.
I agree there is a contradiction here with #440 and in that case #440 is right.
I'll remove the sentence entirely.